src/celeritas/field/RZMapFieldInput.hh

```diff
@@ -29,9 +30,22 @@ struct RZMapFieldInput
     unsigned int num_grid_z{};
     unsigned int num_grid_r{};
     double delta_grid{};  //!< Grid spacing [cm]
-    double offset_z{};  //!< Lower z coordinate [cm]
+    double offset_z{};  //!< Negative offset of the lower z coordinate [cm]
```

Comment:
Declaring it as a "negative offset" seems a little backward to me... I see now that's how it's used in the code but when you're doing linear interpolation isn't it more natural to subtract the lower coordinate rather than add it?

Reply:
Okay, I just remove Negative as it seems contradict to the actual meaning, but did not change the interpolation code as we plan to use UniformGrid and TwodGridCalculator.